packages/frontend/src/index.js

```diff
@@ -10,15 +10,20 @@ import { createStore } from 'redux';
 
 import Routing from './components/Routing';
 import { RECAPTCHA_ENTERPRISE_SITE_KEY } from './config';
-import createRootReducer from './redux/combineReducers';
+import createRootReducer from './redux/createReducers';
 import createMiddleware from './redux/middleware';
 import { initSentry } from './utils/sentry';
+import { wallet } from './utils/wallet';
 
 initSentry();
 
 const history = createBrowserHistory();
 
-export const store = createStore(createRootReducer(history), createMiddleware(history));
+export const store = createStore(createRootReducer(history, wallet.accountId), createMiddleware(history));
```

Comment:
So If we want to pass `accountId` as a param (and don't use `wallet.accountId` inside `setupAccountReducer`) then we need to pass it also here because while creating a store we are adding an initial state for every account that is available, and also we are setting active reducer for an active account. So if we will not pass the accountId here, we will not have an active reducer for the active account, and there will be no data saved by any action.